### ISSUE TITLE
App crashes when I click on a pod

### ISSUE THREAD

**Issue by LucFauvel** [lensapp/lens#2271]
**Describe the bug**
The app crashes when I click on a pod

**To Reproduce**
1. Go to Workloads
2. Click on Pods
3. Select a pod

**Expected behavior**
App not to crash when I click on a pod

**Screenshots**
<img width="1601" alt="Screen Shot 2021-03-02 at 11 00 34 AM" src="https://user-images.githubusercontent.com/14126802/109676577-d536d600-7b46-11eb-8cad-4b9f19501c57.png">

**Environment (please complete the following information):**
- Lens Version: 4.1.3
- OS: macOS Big Sur
- Installation method (e.g. snap or AppImage in Linux): Provided DMG

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
Can't seem to obtai them

**Kubeconfig:**
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: REDACTED
    server: https://REDACTED:443
  name: staging
- cluster:
    certificate-authority-data: REDACTED
    server: REDACTED
  name: REDACTED
contexts:
- context:
    cluster: staging
    user: REDACTED
  name: staging
- context:
    cluster: REDACTED
    user: REDACTED
  name: REDACTED
current-context: staging
kind: Config
preferences: {}
users:
- name: REDACTED
  user:
    client-certificate-data: REDACTED
    client-key-data: REDACTED
    token:  REDACTED
- name: clusterUser_k8s-staging_staging
  user:
    auth-provider:
      config:
        access-token: REDACTED
        apiserver-id: REDACTED
        client-id: REDACTED
        config-mode: "0"
        environment: AzurePublicCloud
        expires-in: "3599"
        expires-on: "1614706588"
        refresh-token: REDACTED
        tenant-id: REDACTED
      name: azure
```

**Comment by hellocoldworld** [comment]
Same error here, 

Environment (please complete the following information):
    Lens Version: 4.1.3
    OS: Ubuntu MATE 18.04
    Installation method (e.g. snap or AppImage in Linux): snapcraft

**Comment by nevalla** [comment]
Duplicate of #2227.

**Comment by hellocoldworld** [comment]
The fix is ready and looks like its going to make it to a patch release soon, so I think we can close this.

**Comment by Nokel81** [comment]
Fixed by #2244 and now in released 4.1.4
